docs/09-01-kubectl-plugin-tutorial.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Use Kyma CLI as kubectl plugin
+type: Tutorials
```

Comment:
Please change the file numeration to 08 as this number corresponds to the tutorial type.

Reply:
Done :)